tests/swe_lun_occult.phpt

```diff
@@ -16,6 +16,9 @@ $tjd_ut = $rv['tret'][0];
 var_dump($tjd_ut, print_date($tjd_ut));
 echo "swe_lun_occult_where for Venus\n";
 var_dump(swe_lun_occult_where($tjd_ut, SE_VENUS, "", SEFLG_SWIEPH));
+echo "swe_lun_occult_when_loc for Venus\n";
+$rv = swe_lun_occult_when_loc(2454466.5, SE_VENUS, "", SEFLG_SWIEPH, 0, 12.1, 49.0, 330);
```

Comment:
The order of the PHP parameters is expected to be:
```
dlsldddl
```
...but the documentation _and_ method call in this test seem to place the last `long` before the last 3 `double`s. At least, I am expecting the `0` parameter to represent a non-backwards (i.e. false) search.

Reply:
Fixed